stdlib/messaging/kafka/src/main/ballerina/src/kafka/producer.bal

```diff
@@ -270,210 +271,246 @@ public type Producer client object {
 function sendStringValues(Producer producer, handle value, handle topic, anydata? key, int? partition, int? timestamp,
     string keySerializerType) returns ProducerError? {
     if (key is ()) {
-        return producerSendString(producer, value, topic, partition, timestamp);
+        return sendString(producer, value, topic, partition, timestamp);
     }
     if (keySerializerType == SER_STRING) {
         if (key is string) {
             handle keyHandle = java:fromString(key);
-            return producerSendStringString(producer, value, topic, keyHandle, partition, timestamp);
+            return sendStringString(producer, value, topic, keyHandle, partition, timestamp);
```

Comment:
Shall we revisit this function names? It bit difficult to get the idea that what this does by looking at the code?

Reply:
IMO,  one of the most important factor is the readability of the code. We should able to read this even in the non-IDE situation. My suggestion is to go further and change to something like
`sendByteArrayValueAndByteArrayKey()` if possible.

Reply:
Agreed. will update.